Realm/ObjectServerTests/RLMSyncTestCase.mm

```diff
@@ -379,27 +379,45 @@ - (void)lazilyInitializeObjectServer {
     if (!self.isParent || s_task) {
         return;
     }
+    NSPipe *pipe = [NSPipe pipe];
+    NSMutableString *stringBuffer = [[NSMutableString alloc] init];
+    __block BOOL inUseError = NO;
     [RLMSyncTestCase runResetObjectServer:YES];
     NSTask *task = [[NSTask alloc] init];
-    task.currentDirectoryPath = [[RLMSyncTestCase rootRealmCocoaURL] path];
+    NSString *currentDirPath = [[[RLMSyncTestCase rootRealmCocoaURL]
+                                 URLByAppendingPathComponent:@"test-ros-instance/"] path];
+    task.currentDirectoryPath = currentDirPath;
     task.launchPath = @"/bin/sh";
-    task.arguments = @[@"build.sh", @"start-object-server"];
+    task.arguments = @[@"-c", @"/usr/local/bin/node ./ros/bin/ros start"];
```

Comment:
Is it necessary to invoke `sh` here? I'd think invoking `/usr/local/bin/node ros/bin/ros start` would be sufficient.